### DISCUSSION TITLE
Let's make main or develop as the default branch?

### DISCUSSION THREAD

**Discussion by akshay5995** [open-sauced/open-sauced#703] [Ideas] [answered]
Hey, Just to be ahead of the curve and show our support to BLM. Let's move to `main` or `develop` as the default branch?

**Comment by NotMoni** [comment]
I agree. but @bdougie might have another idea.

**Comment by JapneetSingh5** [comment]
I think ‘prime’ sounds the coolest

**Comment by bdougie** [comment, marked as answer]
#742 is done and #802 merged.
